src/wallet/legacywallet.cpp

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "wallet/legacywallet.h"
+
+#include "main.h"
+#include "util.h"
+#include "utilmoneystr.h"
+#include "validationinterface.h"
+
+#include <boost/filesystem.hpp>
+#include <boost/thread.hpp>
+
+CWallet* pwalletMain = NULL;
+
+const static std::string DEFAULT_WALLET_FILE = "wallet.dat";
+
+namespace CLegacyWalletModule {
+
+std::string GetWalletFile()
+{
+    return GetArg("-wallet", DEFAULT_WALLET_FILE);
```

Comment:
Not that it would matter that much, but how about using a static variable, that would allow to use ``GetWalletFile()`` without the need to store it into another std::string when using the function, which happens quite a few times.